edx_proctoring/utils.py

```diff
@@ -273,3 +269,17 @@ def verify_and_add_wait_deadline(context, exam, user_id):
         )
         return True
     return False
+
+
+def is_reattempting_exam(from_status, to_status):
+    """
+    Returns a boolean representing whether or not a user is trying to reattempt an exam.
+
+    Given user's exam is in progress, and he is kicked out due to low bandwidth or
+    closes the secure browser. Its expected that the user should not be able to restart
+    the exam workflow.
+    This behavior is being implemented due to edX's integrity constraints.
+    """
+    return ProctoredExamStudentAttemptStatus.is_in_progress_status(from_status) and (
+        ProctoredExamStudentAttemptStatus.is_pre_started_status(to_status)
```

Comment:
redundant parentheses. We can either:
1. Move both statements inside parentheses 
2. remove them